src/client/SvgUtils.ts

```diff
@@ -20,6 +20,29 @@ type BlissaryMapEntryType = {
   blissSvgBuilderCode: string
 };
 
+// Ranges and list for all the indicator symbols.  The range values are the
+// minimum and maximum BCI AV ID.
+const indicatorIds = {
```

Comment:
There are many indicators. Can you explain why these indicators are selected and grouped in this way? What do the terms "rangeN" and "list" mean?

Some indicators such as "25458" and "28046" are marked as "OLD" in the Blissary dictionary, which means they are no longer in use. Why are they still selected?

Can you add a code comment following each array to explain what these indicators represent in English to improve the readability. Thanks.

Reply:
Resolved in a discussion. The decision is to use ranges as is.